api/v1alpha2/virtualmachine_types.go

```diff
@@ -448,7 +448,7 @@ type VirtualMachineAdvancedSpec struct {
 	// VMware Data Recovery.
 	//
 	// +optional
-	ChangeBlockTracking bool `json:"changeBlockTracking,omitempty"`
+	ChangeBlockTracking *bool `json:"changeBlockTracking,omitempty"`
```

Comment:
any implications to making v1a2 changes now?. Are we only trying to prevent additions/removals from older apis after we've made v1a3 the storage version.

Reply:
It doesn't change the on the wire API but, yes, it will result in a compile error. @akutz and I talked about this for https://github.com/vmware-tanzu/vm-operator/pull/484 which is basically the same situation here - we'll take the compile error to avoid a lot of mess when trying to deal with conversion issues.